documentation/modules/ROOT/pages/transformations/mongodb-outbox-event-router.adoc

```diff
@@ -0,0 +1,430 @@
+:page-aliases: configuration/mongodb-outbox-event-router.adoc
+// Category: debezium-using
+// Type: assembly
+// ModuleID: configuring-debezium-connectors-to-use-the-outbox-pattern
+// Title: Configuring {prodname} connectors to use the outbox pattern
+[id="mongodb-outbox-event-router"]
+= MongoDB Outbox Event Router
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+[NOTE]
+====
+This SMT is supported only for the MongoDB connector.
+See {link-prefix}:{link-outbox-event-router}[Outbox Event Router] for the relational database equivalent to this SMT.
+====
+
+The outbox pattern is a way to safely and reliably exchange data between multiple (micro) services. An outbox pattern implementation avoids inconsistencies between a service's internal state (as typically persisted in its database) and state in events consumed by services that need the same data.
+
+To implement the outbox pattern in a {prodname} application, configure a {prodname} connector to:
+
+* Capture changes in an outbox collection
+* Apply the {prodname} MongoDB outbox event router single message transformation (SMT)
+
+A {prodname} connector that is configured to apply the MongoDB outbox SMT should capture changes that occur in an outbox collection only.
+For more information, see {link-prefix}:{link-mongodb-outbox-event-router}#options-for-applying-the-transformation-selectively[Options for applying the transformation selectively].
+
+A connector can capture changes in more than one outbox collection only if each outbox collection has the same structure.
+
+[NOTE]
+====
+This SMT might require multi-document transactions, which have been being supported since MongoDB 4.0, if you need to atomically update a document in a separate collection. For future update, to enable updating existing data and inserting outbox event in an ACID transaction without multi-document transactions, we are planned to support additional configurations for storing outbox events in a form of a sub-document of the existing collection, rather than an independent outbox collection.
```

Comment:
```suggestion
This SMT might require multi-document transactions, which have been being supported since MongoDB 4.0, if you need to atomically update a document in a separate collection. For future update, to enable updating existing data and inserting outbox event in an ACID transaction without multi-document transactions, we have planned to support additional configurations for storing outbox events in a form of a sub-document of the existing collection, rather than an independent outbox collection.
```

Reply:
Also let's describe it more authoritively: "To use this SMT, operations on the actual business collection(s) and the insert into the outbox collection must be done as part of a multi-document transaction.